Source/Magick.NET/Shared/Profiles/Exif/ExifDefinition.cs

```diff
@@ -0,0 +1,364 @@
+﻿using System.Collections;
```

Comment:
It seems that this file is missing the coding header. Could you please add this?

Reply:
Done, in next push.